### PR TITLE
Update version number to 6.9 Beta for the upstream guides

### PR DESCRIPTION
**Pull request by suyogsainkar**
Update attributes.adoc and docinfo files for 6.9
https://bugzilla.redhat.com/show_bug.cgi?id=1918485

### CODE REVIEW

**Comment by melcorr** [comment]
Please learn to rebase rather than merge master. 
If you don't know how, @spetrosi  has a doc prepared with the process.

**Review by spetrosi — CHANGES_REQUESTED** [review]
Let's meet today and I will show you how to rebase. 
The master branch runs a link checker on each PR, and it will fail if you change the ProductVersion attribute to 6.9-beta because these docs are not published. Therefore, this change should be done to the SATELLITE-6.9-beta branch.

File: guides/common/attributes.adoc [hunk 1]
```diff
@@ -5,13 +5,13 @@
 // Versions used in text and code
 :ProjectVersion: 2.3
 :KatelloVersion: 3.18
-:TargetVersion: 6.8
-:TargetVersionMaintainUpgrade: 6.8
+:TargetVersion: 6.9 Beta
```
Comment (spetrosi) [inline guides/common/attributes.adoc:8]: Refer to the attributes in the 6.8-beta branch downstream for the values. The values there are the following:
:TargetVersion: 6.8-beta
:ProductVersion: 6.8-beta
:ProductVersionRepoTitle: Beta

The ProductVersion attribute is used in links, it cannot have spaces there.

**Comment by suyogsainkar** [comment]
@melcorr ack. I only resolved the merge conflict (using the github UI) - which looks like did a merge master instead of a new commit. Not something I expected and would typically do.

@spetrosi sounds good. I wasn't aware this update should be done on the version (SATELLITE-6.9-beta) branch only.

File: guides/common/attributes.adoc (hunk 1 above)
Comment (suyogsainkar) [reply guides/common/attributes.adoc:8]: Ack

**Comment by suyogsainkar** [comment]
As discussed with @spetrosi, closing this one out. Created new PRs below:
https://github.com/theforeman/foreman-documentation/pull/381 (master)
https://github.com/theforeman/foreman-documentation/pull/379 (base SATELLITE-6.9-beta for attributes.adoc)

**Pull request closed without merging**